test/e2e/network/netpol/network_legacy.go

```diff
@@ -776,15 +776,15 @@ var _ = common.SIGDescribe("NetworkPolicyLegacy [LinuxOnly]", func() {
 			framework.ExpectNoError(err, "Error creating Network Policy %v: %v", policy.ObjectMeta.Name, err)
 
 			testCanConnect(f, f.Namespace, "client-a", service, clientAAllowedPort)
-			e2epod.WaitForPodNotFoundInNamespace(f.ClientSet, "client-a", f.Namespace.Name, framework.PodDeleteTimeout)
+			e2epod.WaitForPodNotFoundInNamespace(f.ClientSet, "client-a", f.Namespace.Name, f.Timeouts.PodDelete)
```

Comment:
The default value of `f.Timeouts.PodDelete` is 5 minutes as the same as `framework.PodDeleteTimeout`.
This looks fine for me.

BTW will we remove `framework.PodDeleteTimeout`?

Reply:
> BTW will we remove `framework.PodDeleteTimeout`?

IIUC we could remove _technically_ after migrating is done, though I cannot give an answer as I'm fairly new here, maybe @johnSchnake has some thoughts?

Reply:
Thanks for your explanation, I see that.
Current change seems good for me then.

/test pull-kubernetes-e2e-gce-storage-snapshot
/lgtm

Reply:
/test pull-kubernetes-e2e-gce-storage-snapshot